README.md

```diff
@@ -1,2 +1,71 @@
+However, C++ dependencies in pure-Python projects are typically quite complicated to handle and maintain.
+Here ROD comes to rescue.
+
+URDF, thanks to native ROS support, is historically the most popular robot description used by the community.
+The main problem of URDF is that it is not a specification, and developers of URDF descriptions might produce models and parsers that do not comply to any standard.
+Luckily, URDF models can be easily converted to SDF[^urdf_to_sdf].
+If the URDF model is not compliant, the process errors with clear messages.
+Furthermore, modern versions of the converter produce a SDF description with standardized [pose semantics][pose_semantics],
+that greatly simplifies the life of downstream developers that do not have to guess the reference frame or pose elements.
+Last but not least, the pose semantics also makes SDF aware of the concept of _frame_ that URDF is missing.
+
+## Features
+
+- Out-of-the-box support of SDFormat specifications [≥ 1.7][sdformat_spec_17]
+- Serialization and deserialization support of SDF files
+- In-memory layout based on `dataclasses`
+- Syntax highlighting and auto-completion
+- Support of programmatic creation of SDF files from Python APIs
+- Transitive support of URDF through conversion to SDF[^urdf_to_sdf]
+- Type validation of elements and attributes
+- Automatic check of missing required elements
+- Based on [`Fatal1ty/mashumaro`][mashumaro] for great serialization and deserialization performance
+
+[mashumaro]: https://github.com/Fatal1ty/mashumaro
+[open_robotics]: https://www.openrobotics.org/
+[pose_semantics]: http://sdformat.org/tutorials?tut=pose_frame_semantics_proposal&cat=pose_semantics_docs&
+[sdformat]: http://sdformat.org/
+[sdformat_python]: http://sdformat.org/tutorials?tut=python_bindings&cat=developers&
+[sdformat_repo]: https://github.com/gazebosim/sdformat
+[sdformat_spec]: http://sdformat.org/spec
+[sdformat_spec_17]: http://sdformat.org/spec?elem=sdf&ver=1.7
+[urdf]: http://wiki.ros.org/urdf
+
+[^urdf_to_sdf]: Conversion can be done either using `ign sdf` included in Ignition Gazebo Fortress, or `gz sdf` included in Gazebo Sim starting from Garden.
+
+## Installation
+
+You can install the project with [`pypa/pip`][pip], preferably in a [virtual environment][venv]:
+
+```bash
+pip install git+https://github.com/ami-iit/rod
+```
+
+[pip]: https://github.com/pypa/pip/
+[venv]: https://docs.python.org/3.8/tutorial/venv.html
+
+## Contributing
+
+Pull requests are welcome. 
+For major changes, please open an issue first to discuss what you would like to change.
+
+## Maintainers
+
+| [<img src="https://github.com/diegoferigo.png" width="40">][df] | [@diegoferigo][df] |
+|:---------------------------------------------------------------:|:------------------:|
+
+[df]: https://github.com/diegoferigo
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
```

Comment:
Can we use BSD-3 instead of MIT? We are trying to use BSD-3 in many AMI repos, and it should not put any more limitation on the use w.r.t. to MIT.

Reply:
Sure, done in https://github.com/ami-iit/rod/pull/1/commits/eeaa1a79cb411b5ddd85b609fcd5fa95afdf894c